docs/SRS-Volere/SRS.tex

```diff
@@ -97,7 +97,27 @@ \subsection{Enterprise Constraints}
 \section{Naming Conventions and Terminology}
 \subsection{Glossary of All Terms, Including Acronyms, Used by Stakeholders
 involved in the Project}
-\lips
+\begin{itemize}
+    \item BAS: Building Automation System.
+    \item BCOS: Beyond Compliance Operating System.
+    \item CMMS: Computerized Maintenance Management System.
+    \item Confined Space: A partially or fully enclosed space, not designed for 
+    continuous human occupancy, which has the potential for atmospheric hazards.
+    \item Controlled Space: A City defined space which has the potential to become a confined space.
+    \item CSE: Confined Space Entry
+    \item Dry Well: A room which houses industrial equipment such as pumps and valves
+    \item Hazard Assessment Form: A form outlining potential hazards at a location
+    \item Hot Works: Work which produces ignition sources. Requires an accompanying hot works permit.
```

Comment:
I think this and the Controlled space line exceed the 80 character line limit

Reply:
fixed